website/src/components/StorybookComponent/index.tsx

```diff
@@ -45,11 +49,20 @@ function joinNameAndVariant(name: string, variant: string | undefined) {
   return [name, variant].filter((t) => !!t).join("--");
 }
 
-function getStoryUrlId(implementation: Implementation, name: string, variant: string | undefined) {
+function getStoryUrlId(
+  implementation: Implementation,
+  name: string,
+  variant: string | undefined,
+  isExamplePage: boolean | undefined,
+) {
   if (isComposedStorybook(implementation)) {
     return `${implementation}_${joinNameAndVariant(name, variant)}`;
   }
 
+  if (isExamplePage) {
+    return `${joinNameAndVariant(name, variant)}`;
```

Comment:
De template literals voegen hier niet zoveel toe:

```suggestion
    return joinNameAndVariant(name, variant);
```

Reply:
Gedaan